content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -27,11 +27,11 @@ multiple scheduling retries will be needed.
 
 ## {{% heading "prerequisites" %}}
 
-Kubernetes v{{< skew currentVersion >}} includes cluster-level API support for
+Kubernetes v{{< skew currentPatchVersion >}} includes cluster-level API support for
 storage capacity tracking. To use this you must also be using a CSI driver that
 supports capacity tracking. Consult the documentation for the CSI drivers that
 you use to find out whether this support is available and, if so, how to use
-it. If you are not running Kubernetes v{{< skew currentVersion >}}, check the
+it. If you are not running Kubernetes v{{< skew currentPatchVersion >}}, check the
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.